inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/settings/WorkflowAndMonitoringPanel.java

```diff
@@ -84,29 +89,39 @@ public WorkflowAndMonitoringPanel(String aID, IModel<Project> aProject)
         form.add(monitoringChoices);
 
         //Finally, add the confirm button at the end
-
-        Button confirm = new AjaxButton(getString("confirm"),Model.of("Confirm")) {
-            @Override
-            public void onSubmit(AjaxRequestTarget target)
-            {
-                if (monitoringChoices.getDefaultModelObjectAsString().equals(monitoring.get(0)))
-                {
-                    workloadProperties.setActive(false);
-                } else {
-                    workloadProperties.setActive(true);
-                }
-
-                if (workflowChoices.getDefaultModelObjectAsString().equals(workflow.get(0)))
-                {
-                    workflowProperties.setActive(false);
-                } else {
-                    workflowProperties.setActive(true);
-                }
-
-                setResponsePage(getApplication().getHomePage());
-            }
-        };
+        Button confirm = new LambdaAjaxButton(getString("confirm"),
+            this::actionConfirm).triggerAfterSubmit();
```

Comment:
Is there a specific reason that triggerAfterSubmit() is used here? If yes, please add a comment.